docs/marionette.features.md

```diff
@@ -6,6 +6,14 @@ Marionette Features are opt-in functionality. That you can enable by setting `Ma
 + make it possible to add breaking changes in a minor release
 + give community members a chance to provide feedback for new functionality
 
+## `childViewEventPrefix`
+
+This flag is set to `true` by default.
+
+This flag indicates whether [`childViewEventPrefix`](./events.md#a-child-views-event-prefix)
+for all views will return the default value of `'childview'` or if it will return `false`
+disabling [automatic event bubbling](./events.md#event-bubbling).
```

Comment:
for better or worse, `childViewEventPrefix` is currently calculated per proxied event so you can flip this switch on-and-off all day long and it'll respect what it is set to at the moment of a `triggerMethod`.

I think this'd be a good thing to set during view instantiation.  both `events` and `triggers` are setting during `delegateEvents`

I think the `childViewEventPrefix` `_.result` should move into `_buildEventProxies` along with the `normalizeMethods` [here](https://github.com/marionettejs/backbone.marionette/blob/master/src/mixins/view.js#L194).  And we should default `_childViewEvents` and  `_childViewTriggers` to `{}` so we can remove the first checks

Honestly I think we should calculate those hashes such that the handler should be able to be as simple as:
```js
_childViewEventHandler(eventName, ...args) {
    if (this._childViewEvents[eventName]) {
      this._childViewEvents[eventName].apply(this, args);
    }

    if (this._childViewTriggers[eventName])) {
      this.triggerMethod(this._childViewTriggers[eventName], ...args);
    }

    if (this._childViewEventPrefix !== false) {
      const childEventName = this._childViewEventPrefix + ':' + eventName;

      this.triggerMethod(childEventName, ...args);
    }
  }
};
```